docs/guides/admin/docs/modules/watsontranscripts.md

```diff
@@ -25,83 +25,106 @@ Workflow 2 runs:
+If _max.attempts_ > 1 and the service receives an error callback, it will start the workflow specified in
+_retry_workflow_ to create a new job. When _max.attempts_ is reached for a track, the service will stop retrying.
 
 ```
 # Change enabled to true to enable this service.
-enabled=true
+enabled=false
+
+# IBM Watson Speech-to-Text service url
+# Default: https://stream.watsonplatform.net/speech-to-text/api
+# ibm.watson.service.url=https://stream.watsonplatform.net/speech-to-text/api
 
-# User obtained when registering with the IBM Watson Speech-to_text service
-ibm.watson.user=<SERVICE_USER>
+# APi key obtained when registering with the IBM Watson Speech-to_text service.
+# If empty, user and password below will be used.
+ibm.watson.api.key=<API_KEY>
+
+# User obtained when registering with the IBM Watson Speech-to_text service.
+# Mandatory if ibm.watson.api.key not entered.
+#ibm.watson.user=<SERVICE_USER>
 
 # Password obtained when registering with the IBM Watson Speech-to_text service
-ibm.watson.password=<SERVICE_PSW>
+# Mandatory if ibm.watson.api.key not entered.
+#ibm.watson.password=<SERVICE_PSW>
 
 # Language model to be used. See the IBM Watson Speech-to-Text service documentation
-# for available models. If empty, the default will be used ("en-US_BroadbandModel").
-#ibm.watson.model=
+# for available models.
+# Default: en-US_BroadbandModel
+#ibm.watson.model=en-US_BroadbandModel
 
 # Workflow to be executed when results are ready to be attached to media package.
-#workflow=attach-watson-transcription
+# Default: attach-watson-transcripts
+#workflow=attach-watson-transcripts
 
 # Interval the workflow dispatcher runs to start workflows to attach transcripts to the media package
-# after the transcription job is completed.
-# (in seconds) Default is 1 minute.
+# after the transcription job is completed. In seconds.
+# Default: 60
 #workflow.dispatch.interval=60
 
 # How long it should wait to check jobs after their start date + track duration has passed.
-# The default is 10 minutes. This is only used if we didn't get a callback from the
-# ibm watson speech-to-text service.
-# (in seconds)
+# This is only used if we didn't get a callback from the ibm watson speech-to-text service.
+# In seconds.
+# Default: 600
 #completion.check.buffer=600
 
 # How long to wait after a transcription is supposed to finish before marking the job as
-# canceled in the database. Default is 2 hours.
-# (in seconds)
+# canceled in the database. In seconds. Default is 2 hours.
+# Default: 7200
 #max.processing.time=7200
 
 # How long to keep result files in the working file repository in days.
-# The default is 7 days.
+# Default: 7
 #cleanup.results.days=7
 
 # Email to send notifications of errors. If not entered, the value from
 # org.opencastproject.admin.email in custom.properties will be used.
 #notification.email=
+
+# Start transcription job load
+# Default: 0.1
+#job.load.start.transcription=0.1
```

Comment:
Dare I say this could be zero.  0.1 is probably a safer value though ;)

Reply:
Does 0 work? :) I always use 0.1 for low-cost jobs.

Reply:
It should, but 0.1 is likely safer.